src/custom-typings/jest.d.ts

```diff
@@ -0,0 +1,30 @@
+// Compiled using typings@0.6.6
+// Source: https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/86dbea8fc37d9473fee465da4f0a21bea4f8cbd9/jest/jest.d.ts
+// Type definitions for Jest 0.1.18
+// Project: http://facebook.github.io/jest/
+// Definitions by: Asana <https://asana.com>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
```

Comment:
we need to take out this comment and explain why we wrote this file instead of using those typings